README.md

```diff
@@ -44,6 +44,22 @@ steps:
   run: ls -R
   working-directory: path/to/artifact
 ```
+
+## Compatibility between `v1` and `v2`
+
+When using `download-artifact@v1`, a directory denoted by the name of the artifact would be created if the `path` input was not provided. All of the contents would be downloaded to this directory.
```

Comment:
Does the existence of `path` input matter?  Is a folder with artifact name always created in `v1`?

Reply:
The `path` does matter, the folder with the name is only created if the `path` is not provided for `v1`. 

`v1` with `path` provided: https://github.com/konradpabjan/artifact-test/runs/634397827?check_suite_focus=true#step:7:15
`v1` with `path` not provided: https://github.com/konradpabjan/artifact-test/runs/634463778?check_suite_focus=true#step:7:17